launch/app/app.bndrun

```diff
@@ -211,4 +212,5 @@ feature.openhab-model-runtime-all: \
 	org.threeten.extra;version='[1.4.0,1.4.1)',\
 	slf4j.api;version='[1.7.25,1.7.26)',\
 	tec.uom.lib.uom-lib-common;version='[1.0.3,1.0.4)',\
-	tec.uom.se;version='[1.0.10,1.0.11)'
+	tec.uom.se;version='[1.0.10,1.0.11)',\
+	slf4j.simple;version='[1.7.21,1.7.22)'
```

Comment:
I don't think slf4j-simple should be used by the OSGi container.
I also don't think slf4j-simple should be part of the dependency tree of any OH OSGi bundles at all.

Reply:
I didn't touch this section, it is automatically filled when doing a "resolve" - so I committed the change in order to avoid people seeing local changes in their IDE.
Are you saying that bnd does NOT add this line, when you run the resolution?

Reply:
The resolve logic is using the index generated by the dependencies.

As written above slf4j-simple should not be part of the index / dependency tree at all. If bnd does not know about a slf4j-simple bundle it cannot add them.

The runtime needs "some" implementation of the slf4j API.
So "resolve" may choose one of all the implementation that are part of the index.
If could be the one this time, another one the next time. It depends on the wiring at all.